core/src/main/java/org/sonarsource/sonarlint/core/analyzer/sensor/ScannerExtensionDictionnary.java

```diff
@@ -77,21 +72,21 @@ public BatchExtensionDictionnary(ComponentContainer componentContainer, DefaultS
     return Phase.Name.DEFAULT;
   }
 
-  private <T> List<T> getFilteredExtensions(Class<T> type, @Nullable Project project, @Nullable ExtensionMatcher matcher) {
+  private <T> List<T> getFilteredExtensions(Class<T> type, @Nullable Project project) {
     List<T> result = new ArrayList<>();
     for (Object extension : getExtensions(type)) {
       if (org.sonar.api.batch.Sensor.class.equals(type) && extension instanceof Sensor) {
         extension = new SensorWrapper((Sensor) extension, sensorContext, sensorOptimizer);
       }
-      if (shouldKeep(type, extension, project, matcher)) {
+      if (shouldKeep(type, extension, project)) {
         result.add((T) extension);
       }
     }
     if (org.sonar.api.batch.Sensor.class.equals(type)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Sensor"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)